CHANGELOG.md

```diff
@@ -7,6 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [python-0.2.0] - Unreleased
+
+### Changed
+
+- `lace.Engine.__init__` now takes a Pandas' or Polars' dataframe to initialize an `Engine`.
+
+### Added
+
+- `Engine.load` supports loading metadata from disk.
+- `lace.CodebookBuilder` supports loading a codebook from disk or using parameterized inference for creating a new `Engine`.
+- Can append new columns to an `Engine.append_columns`
+    + Specify the types of new columns with `ColumnMetadata`
+- Can delete columns using `Engine.del_column`
```

Comment:
It looks like there is a new method in `lace.plot` called `prediction_uncertainty`